### ISSUE TITLE
ERROR: A failure occurred in check()

### ISSUE THREAD

**Issue by openmindead** [tpm2-software/tpm2-tss#1240]
Good day, I'm experiencing this during the make check-TESTS stage:

```
...
ERROR: test/integration/sapi-encrypt-decrypt.int
...
ERROR: test/integration/esys-createloaded-session.int
...
============================================================================
Testsuite summary for tpm2-tss 2.1.0
============================================================================
# TOTAL: 125
# PASS:  121
# SKIP:  2
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 2
============================================================================
See ./test-suite.log
Please report to https://github.com/tpm2-software/tpm2-tss/issues
============================================================================
```

Here is test-suite.log:
```
======================================
   tpm2-tss 2.1.0: ./test-suite.log
======================================

# TOTAL: 125
# PASS:  121
# SKIP:  2
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 2

.. contents:: :depth: 2

ERROR: test/integration/sapi-encrypt-decrypt
============================================

Starting simulator on port 44112
successfully started daemon: tpm_server with PID: 11044
/tmp/trizen-openm/tpm2-tss/src/tpm2-tss-2.1.0
simulator PID: 11044
Simulator with PID 11044 bound to port 44112 and  44113 successfully.
WARNING:tcti:src/util/io.c:248:socket_connect() Failed to connect to host 127.0.0.1, port 44112: errno 111: Connection refused 
Failed to initialize tcti context: 0xa000a
TPM_StartUp failed
ERROR test/integration/sapi-encrypt-decrypt.int (exit status: 99)

ERROR: test/integration/esys-createloaded-session
=================================================

Starting simulator on port 59304
successfully started daemon: tpm_server with PID: 19565
/tmp/trizen-openm/tpm2-tss/src/tpm2-tss-2.1.0
simulator PID: 19565
Simulator with PID 19565 bound to port 59304 and  59305 successfully.
WARNING:tcti:src/util/io.c:248:socket_connect() Failed to connect to host 127.0.0.1, port 59304: errno 111: Connection refused 
Failed to initialize tcti context: 0xa000a
TPM_StartUp failed
ERROR test/integration/esys-createloaded-session.int (exit status: 99)

SKIP: test/integration/esys-field-upgrade
=========================================

Starting simulator on port 10418
successfully started daemon: tpm_server with PID: 20925
/tmp/trizen-openm/tpm2-tss/src/tpm2-tss-2.1.0
simulator PID: 20925
Simulator with PID 20925 bound to port 10418 and  10419 successfully.
Execute the test script
WARNING:esys:src/tss2-esys/api/Esys_FieldUpgradeData.c:300:Esys_FieldUpgradeData_Finish() Received TPM Error 
ERROR:esys:src/tss2-esys/api/Esys_FieldUpgradeData.c:107:Esys_FieldUpgradeData() Esys Finish ErrorCode (0x00000143) 
Script returned 77
SKIP test/integration/esys-field-upgrade.int (exit status: 77)

SKIP: test/integration/esys-firmware-read
=========================================

Starting simulator on port 53432
successfully started daemon: tpm_server with PID: 21025
/tmp/trizen-openm/tpm2-tss/src/tpm2-tss-2.1.0
simulator PID: 21025
Simulator with PID 21025 bound to port 53432 and  53433 successfully.
Execute the test script
WARNING:esys:src/tss2-esys/api/Esys_FirmwareRead.c:283:Esys_FirmwareRead_Finish() Received TPM Error 
ERROR:esys:src/tss2-esys/api/Esys_FirmwareRead.c:99:Esys_FirmwareRead() Esys Finish ErrorCode (0x00000143) 
Script returned 77
SKIP test/integration/esys-firmware-read.int (exit status: 77)
```
System: Manjaro Linux x64
PKGBUILD: https://aur.archlinux.org/packages/tpm2-tss

**Comment by openmindead** [comment]
OK for those who may face the same issue a solutions is to add yourself to tss group.

**Comment by diabonas** [comment]
@openmindead Adding yourself to the `tss` shouldn't be necessary for the tests to pass (even though you might want to to it anyway in order to use the TPM without root). The ERRORs are just intermittent failures, usually simply trying to build the package a second time causes the tests to pass. This is a known problem, see #1227, which should be fixed on master with PR #1234 (which you can build with the [tpm2-tss-git](https://aur.archlinux.org/packages/tpm2-tss-git) AUR package), but hasn't yet made it into a released version.
